pyinstrument/middleware.py

```diff
@@ -41,8 +61,10 @@ def process_response(self, request, response):
         if hasattr(request, "profiler"):
             profile_session = request.profiler.stop()
 
-            renderer = HTMLRenderer()
-            output_html = renderer.render(profile_session)
+            configured_renderer = getattr(settings, "PYINSTRUMENT_PROFILE_DIR_RENDERER", None)
```

Comment:
Keeping this here since  this renderer is used in the `if/else` block to produce output.